tests/blame/buffer.c

```diff
@@ -17,6 +17,28 @@ void test_blame_buffer__cleanup(void)
 	git_repository_free(g_repo);
 }
 
+void test_blame_buffer__index(void)
+{
+	const git_blame_hunk *hunk;
+	const char *buffer = "Hello\nWorld!";
+
+	git_blame_free(g_fileblame);
+	g_fileblame = NULL;
```

Comment:
These two lines shouldn't be needed, we already clean up `g_fileblame` in `test_blame_buffer__cleanup`.

Reply:
ASan complains if I remove them. this is because this is executed after `test_blame_buffer__initialize()`, so if I remove them, I would be leaking some stuff from the previously opened `g_fileblame`.

let me add a comment to avoid confusion.

Reply:
Comment added!